include/openssl/aead.h

```diff
@@ -15,6 +15,7 @@
 #ifndef OPENSSL_HEADER_AEAD_H
 #define OPENSSL_HEADER_AEAD_H
 
+#include <openssl/aes.h>
```

Comment:
Redundant now?

Reply:
The best thing about this code review is how there are more comments than changed/added lines. ;-)

Right you are!